lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -56,6 +56,68 @@ removeNode = (node) ->
   removedNode = node.parentNode.removeChild(node)
   triggerEvent('page:after-node-removed', removedNode)
 
+updateHead = (doc) ->
+  loadedAssets = extractTrackAssets document
+  fetchedAssets = extractTrackAssets doc
+
+  updateLinkTags(loadedAssets, fetchedAssets)
+  updateScriptTags(loadedAssets, fetchedAssets)
+
+extractTrackAssets = (doc) ->
+  for node in doc.querySelector('head').childNodes when node.getAttribute?('data-turbolinks-track')?
+    node
+
+filterForNodeType = (nodeList, nodeType) ->
+  (node for node in nodeList when node.nodeName == nodeType)
+
+insertScript = (scriptNode) ->
+  newNode = document.createElement 'script'
+  newNode.setAttribute attr.name, attr.value for attr in scriptNode.attributes
+  newNode.appendChild document.createTextNode scriptNode.innerHTML
+
+  deferred = $.Deferred()
```

Comment:
we already have a promise polyfill in the admin, do we use turbograft anywhere else?